core/indexer/elasticsearch.go

```diff
@@ -223,6 +237,72 @@ func (ei *elasticIndexer) SaveBlock(
 	}
 }
 
+// SaveRoundInfo will save on elastic search information about round
+func (ei *elasticIndexer) SaveRoundInfo(round int64, signersIndexes []uint64) {
+	var buff bytes.Buffer
+
+	serializedSignersIndexes, err := ei.marshalizer.Marshal(SignersIndexes{SignersIndexes: signersIndexes})
```

Comment:
_marshalized_ over _serialized_ ? as we use this naming everywhere

Reply:
Also move the struct declaration and initialization above this line and maybe split in more lines, not one liner.

Reply:
still not moved the declaration part

Reply:
Done.